engine/channels.go

```diff
@@ -190,13 +192,15 @@ func initializeChannelRoleMap() {
 		flow.RoleAccess}
 	channelRoleMap[ReceiveApprovals] = flow.RoleList{flow.RoleConsensus, flow.RoleVerification}
 
-	channelRoleMap[ProvideCollections] = flow.RoleList{flow.RoleCollection, flow.RoleExecution}
+	channelRoleMap[ProvideCollections] = flow.RoleList{flow.RoleCollection, flow.RoleExecution, flow.RoleAccess}
 	channelRoleMap[ProvideChunks] = flow.RoleList{flow.RoleExecution, flow.RoleVerification}
 	channelRoleMap[ProvideReceiptsByBlockID] = flow.RoleList{flow.RoleConsensus, flow.RoleExecution}
 	channelRoleMap[ProvideApprovalsByChunk] = flow.RoleList{flow.RoleConsensus, flow.RoleVerification}
+}
 
-	channelRoleMap[syncClusterPrefix] = flow.RoleList{flow.RoleCollection}
-	channelRoleMap[consensusClusterPrefix] = flow.RoleList{flow.RoleCollection}
```

Comment:
It's arguably OK to remove this and replace it by the `ClusterChannelRoles` logic, but the issue is that there are tests, such as `TestUniqueChannels_ClusterChannels`, that still rely on these prefixes being present in the `channelRoleMap`.

Reply:
Updated